src/main/java/cms/gov/madie/measure/services/MeasureService.java

```diff
@@ -1,5 +1,6 @@
 package cms.gov.madie.measure.services;
 
+import com.okta.commons.lang.Collections;
```

Comment:
Is the okta collections lib preferred here?

Reply:
whoops. I just did whatever intellij suggested...bad intellij

Reply:
I thought java.util had the isEmpty check, but I was wrong so I switched it to springs.